media-gfx/qview/qview-3.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Minimalist image viewer"
+DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
```

Comment:
The homepage has some more info about the app's main features. You may want to add those to the description. Might give potential users a reason to try qview,